src/git.py

```diff
@@ -53,7 +53,7 @@ def get_changed_files(self):
 
         for file in files:
             result = re.match(
-                r"models/(\w)+/(\w+)+.sql", file.get("filename"), flags=re.IGNORECASE
+                r"models/(.*/)*(\w+)+.sql", file.get("filename"), flags=re.IGNORECASE
```

Comment:
```suggestion
                r"models/.*/\w+.sql", file.get("filename"), flags=re.IGNORECASE
```

This matches the same text, but is a simpler regex

Reply:
The edge case is `models//filename.sql` - not sure if that's valid, but it wouldn't be matched by the suggested regex.

Reply:
I don't think empty path segments are valid, so this should be fine